pkg/miscellaneous/miscellaneous_test.go

```diff
@@ -343,9 +343,10 @@ var _ = Describe("Miscellaneous Tests", func() {
 			statefulSetName = "etcd-test"
 			podName         = "etcd-test-0"
 			namespace       = "test_namespace"
+			emptyString     = ""
 		)
 		Context("In single-node etcd", func() {
-			It("Should return the cluster state as `new` ", func() {
+			It("Should return the cluster state as empty string ", func() {
```

Comment:
Shouldn't the cluster state still be `new` for this case?

Reply:
Ideally this test case should be removed as I added this when we are depending on [GetInitialClusterState](https://github.com/gardener/etcd-backup-restore/blob/master/pkg/miscellaneous/miscellaneous.go#L449) to get state but now it will only determine state for scale-up scenario and I have also  rename GetInitialClusterState to `GetInitialClusterStateIfScaleup` to avoid ambiguity.

Reply:
Ok. In that case, can you please adapt these tests too? I would like to avoid removing existing test cases if possible.

Reply:
done.